src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -401,6 +429,11 @@ private ErrorRecord NewError()
         /// <returns>An object suitable for serializing to JSON</returns>
         private object ProcessValue(object obj, int depth)
         {
+            if (_stopping)
```

Comment:
hmm perhaps instead of repeating this, add a `IsStopping()` method. That way if, for whatever reason, we need to implement this in more places or change/expand the stop actions we can do it in one place.